src/shared/components/views/DatatableWithToggle.tsx

```diff
@@ -8,13 +15,14 @@ import { sendGtagEventFeatureDataTableViewClick } from '../../utils/gtagEvents';
 
 import './styles/datatable-toggle.scss';
 
-const DatatableWithToggle: FC = ({ children }) => {
+const DatatableWithToggle = ({ children }: { children: ReactNode }) => {
```

Comment:
Just removing `FC` now because future versions of React won't have this type available